Update tableau.md

rpourzand opened the pull request:
Adding mention of entities, saved queries, and calculated field updates


## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If related to an open 
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment when publishing docs for a prerelease version of dbt:
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

rpourzand committed:
Update tableau.md

rpourzand committed:
Update tableau.md

rpourzand commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -59,12 +59,14 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
-   
 - Certain Table calculations like "Totals" and "Percent Of" may not be accurate when using metrics aggregated in a non-additive way (such as count distinct)
 - In any of our Semantic Layer interfaces (not only Tableau), you must include a [time dimension](/docs/build/cumulative#limitations) when working with any cumulative metric that has a time window or granularity.
+- We support calculated fields to the extent that you are using them to create a parameter filter or dynamically select metrics and dimensions, but other calculated field functions are not supported.
```
This sounds super awkward. @aiguofer trying to figure out if I've captured the gist of it.

mirnawong1 replied:
what about:

> We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported.

aiguofer replied:
I like Mirna's suggestion but I would change `calculated field functions` to `uses of calculated fields`.

I wonder if we want to mention something like "if you have use cases for calculated fields that are not currently covered, please reach out so we understand the use case to see if we can support it".

There's probably a lot of use cases that we're not even aware of that might be totally legitimate and doable.

rpourzand replied:
That's a good idea! @mirnawong1 the "Things to note section" is getting kind of large. I'm going to try and break it up a bit and add these changes

mirnawong1 committed:
Merge branch 'current' into rpourzand-some-tableau-updates

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -45,9 +45,9 @@ This integration provides a live connection to the dbt Semantic Layer through Ta
 ## Using the integration
 
-1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page with all the metrics and dimensions configured in your dbt Semantic Layer. 
+1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to chooset between all the metrics, dimensions, entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](https://docs.getdbt.com/docs/build/saved-queries) that you've defined. These will also show up  ...
```
```suggestion
1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to choose between all the metrics, dimensions, and entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](https://docs.getdbt.com/docs/build/saved-queries) that you've defined. These will also show up as data sources when you log in with the connector.
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -45,9 +45,9 @@ This integration provides a live connection to the dbt Semantic Layer through Ta
 ## Using the integration
 
-1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page with all the metrics and dimensions configured in your dbt Semantic Layer. 
+1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to choose between all the metrics, dimensions, and entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](https://docs.getdbt.com/docs/build/saved-queries) that you've defined. These will also show  ...
```
```suggestion
1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to choose between all the metrics, dimensions, and entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](/docs/build/saved-queries) that you've defined. These will also show up as data sources when you log in with the connector.
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -45,9 +45,9 @@ This integration provides a live connection to the dbt Semantic Layer through Ta
+1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to choose between all the metrics, dimensions, and entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](/docs/build/saved-queries) that you've defined. These will also show up as data sources when ...
 2. **Access worksheet** &mdash; From there, go directly to a worksheet in the bottom left-hand corner.
-3. **Access metrics and dimensions** &mdash; Then, you'll find all the metrics and dimensions that are available to query on the left side of your window.
+3. **Access metrics and dimensions** &mdash; Then, you'll find all the metrics, dimensions, entities that are available to query on the left side of your window.
```
```suggestion
3. **Access metrics and dimensions** &mdash; Then, you'll find all the metrics, dimensions, and entities that are available to query on the left side of your window.
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -59,12 +59,14 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
 
 - All metrics use the "SUM" aggregation type, and this can't be altered. The dbt Semantic Layer controls the aggregation type and it is intentionally fixed. Keep in mind that the underlying aggregation in the dbt Semantic Layer might not be "SUM" (even though "SUM" is Tableau's default).
-- Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). 
+-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). For smaller slices of data that can be joined together, you can access saved queries.
```
```suggestion
-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). You can use saved queries for smaller pieces of data that you want to combine.
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 committed:
Merge branch 'current' into rpourzand-some-tableau-updates

mirnawong1 commented:
hey @rpourzand - reviewed it but left a sugg and waiting to hear what @aiguofer says! 🙏

aiguofer commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -59,12 +59,14 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
 
 - All metrics use the "SUM" aggregation type, and this can't be altered. The dbt Semantic Layer controls the aggregation type and it is intentionally fixed. Keep in mind that the underlying aggregation in the dbt Semantic Layer might not be "SUM" (even though "SUM" is Tableau's default).
-- Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). 
+-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). You can use saved queries for smaller pieces of data that you want to combine.
```
Might be worth mentioning that filters from saved queries will be applied to all queries using it.

rpourzand committed:
Update tableau.md

Updating with suggestions and other info

rpourzand committed:
Update tableau.md

Fixing a typo and clarifying some language

rpourzand commented:
@mirnawong1 @aiguofer I updated some more information here based on suggestions and broke out the "Things to note" Section - let me know your thoughts

@aiguofer  I also added the note about ALL vs METRICS AND DIMENSIONS

mirnawong1 committed:
Merge branch 'current' into rpourzand-some-tableau-updates

mirnawong1 committed:
Merge branch 'current' into rpourzand-some-tableau-updates

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -56,15 +58,21 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
 
 
-## Things to note
+## Things to Note
```
sentence case to match our style guide 🙏 
```suggestion
## Things to note
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -58,13 +60,19 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
+**Aggregation**
+All metrics are shown as using the "SUM" aggregation type in Tableau's UI, and this cannot be altered using Tableau's interface. The dbt Semantic Layer controls the aggregation type in code and it is intentionally fixed. Keep in mind that the underlying aggregation in the dbt Semantic Layer might not be "SUM" ("SUM" is Tableau's default).
+
+### Data Sources and Display
```
```suggestion
### Data sources and display
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -58,13 +60,19 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
+-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). You can use saved queries for smaller pieces of data that you want to combine.
+- To display available metrics and dimensions, dbt Semantic Layer returns metadata for a fake table with the dimensions and metrics as 'columns' on this table. Because of this, you can't actually query this table for previews or extracts.
+
+### Calculations & Querying
```
```suggestion
### Calculations and querying
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -58,13 +60,19 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
+### Calculations and querying
 - Certain Table calculations like "Totals" and "Percent Of" may not be accurate when using metrics aggregated in a non-additive way (such as count distinct)
 - In any of our Semantic Layer interfaces (not only Tableau), you must include a [time dimension](/docs/build/cumulative#limitations) when working with any cumulative metric that has a time window or granularity.
+- We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported. _Note: If you have use cases for calculated fields that are not currently covered, please reach out to dbt Support we understand the use case_
```
```suggestion
- We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported. _Note: For calculated fields use cases that are not currently covered, please reach out to [dbt Support](mailto:support@getdbt.com) and share them so we can further understand._
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 commented on website/docs/docs/use-dbt-semantic-layer/tableau.md:
```diff
@@ -58,13 +60,19 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
+### Calculations and querying
 - Certain Table calculations like "Totals" and "Percent Of" may not be accurate when using metrics aggregated in a non-additive way (such as count distinct)
 - In any of our Semantic Layer interfaces (not only Tableau), you must include a [time dimension](/docs/build/cumulative#limitations) when working with any cumulative metric that has a time window or granularity.
+- We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported. _Note: For calculated fields use cases that are not currently covered, please reach out to [dbt Support](mailto:support@getdbt.com) and share them so we can further understand._
```
```suggestion
- We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other uses of calculated fields are not supported. _Note: For calculated fields use cases that are not currently covered, please reach out to [dbt Support](mailto:support@getdbt.com) and share them so we can further understand._
```

mirnawong1 committed:
Update website/docs/docs/use-dbt-semantic-layer/tableau.md

mirnawong1 reviewed (approved):
hey @rpourzand and @aiguofer - this looks good to me! are we ok to merge?

mirnawong1 committed:
Merge branch 'current' into rpourzand-some-tableau-updates

Commit:
fixing heder

Commit:
Merge branch 'rpourzand-some-tableau-updates' of https://github.com/dbt-labs/docs.getdbt.com into rpourzand-some-tableau-updates

mirnawong1 commented:
hey team, fixed up the topics under 'things to note' - is this good to go?

![Screenshot 2024-01-30 at 11 10 14](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/a7f0ec00-0325-4384-91c5-8a5b79c9f512)

mirnawong1 committed:
Merge branch 'current' into rpourzand-some-tableau-updates
